### PR TITLE
Reverted class.avatar and upload image

### PR DESCRIPTION
**Pull request by TheStrato**
Reverted /include/class.avatar
Uploaded an image to /images

### CODE REVIEW

**Comment by TheStrato** [comment]
I am still learning. But what all this boils down to is I went back to the original class.avatar, then did a PR to add the image to the repo, then put the image into the FTP so I could generate a URL, then I did another PR with that URL in the class.avatar.

File: include/class.avatar.php [hunk 1]
```diff
@@ -211,6 +211,8 @@ class Gravatar
     function __construct($user, $imageset) {
         $this->email = $user->getEmail();
         $this->d = $imageset;
+        // Changed for new default avatar
++        $this->d = 'https://support.stage-9.co.uk/images/stage9projectpatch.png'
```
Comment (dapperrobotbear) [inline include/class.avatar.php]: You've accidentally added a "+" at the beginning of this line.

**Comment by HPringles** [comment]
Plus it isn't URL encoded - the other PR is now up to date and should work - reccomend this one is closed

**Comment by TheStrato** [comment]
Apparently I did it wrong anyway. So I will just close this one.

**Pull request closed without merging**